packages/big-bang-core/README.md

```diff
@@ -18,34 +22,33 @@ This package deploys [Big Bang Core](https://repo1.dso.mil/platform-one/big-bang
 # Clone the binaries
 git clone https://github.com/defenseunicorns/zarf.git
 
-# Change dir
-cd zarf
+# change to the examples folder
+cd zarf/examples/big-bang-core
```

Comment:
Agree with making this sit in examples, but can we move the whole dir to `examples/big-bang` vs `packages/big-bang-core`?